### PR TITLE
[MenuBundle] fix last PR mistakes in the menuBundle and corrected styling issues f…

### PR DESCRIPTION
**Pull request by denbatte**
…rom squashing

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets |

Squashing the last PR made a mess of the code formatting. This PR should correct this.
There was also a bug in the code that made it impossible to get menu items. My mistake.

### CODE REVIEW

File: src/Kunstmaan/MenuBundle/AdminList/MenuItemAdminListConfigurator.php [hunk 1]
```diff
@@ -38,7 +38,6 @@ public function __construct(EntityManager $em, AclHelper $aclHelper = null, Menu
     public function buildFields()
     {
         $this->addField('title', 'Title', false, 'KunstmaanMenuBundle:AdminList:menu-item-title.html.twig');
-        $this->addField('online', 'Online', false, 'KunstmaanMenuBundle:AdminList:menu-item-online.html.twig');
```
Comment (jockri) [inline src/Kunstmaan/MenuBundle/AdminList/MenuItemAdminListConfigurator.php]: Why did your remove this?

**Comment by roderik** [comment]
@denbatte merge conflict

File: src/Kunstmaan/MenuBundle/AdminList/MenuItemAdminListConfigurator.php (hunk 1 above)
Comment (aistis-) [inline src/Kunstmaan/MenuBundle/AdminList/MenuItemAdminListConfigurator.php]: I guess it is because there is no `online` field in `MenuItem`

Comment (denbatte) [inline src/Kunstmaan/MenuBundle/AdminList/MenuItemAdminListConfigurator.php]: In my opinion it is confusing to have a online / offline value for an external link.
Also, I feel that the online offline value should only be shown on the pages and not on the menu.

Comment (jockri) [inline src/Kunstmaan/MenuBundle/AdminList/MenuItemAdminListConfigurator.php]: An internal menu item will not be shown on the website when the page is offline. So I thinks it is handy to have an indication in the list of menu items, so you directly see the reason why the link is not showing (especially for people with an non-technical background). Perhaps we can leave the column empty for external links if that is the problem..

Comment (denbatte) [inline src/Kunstmaan/MenuBundle/AdminList/MenuItemAdminListConfigurator.php]: Yes, a good plan. I will alter the PR.

**Comment by aistis-** [comment]
+1

**Pull request merged**